teuthology/dispatcher/supervisor.py

```diff
@@ -269,7 +269,7 @@ def unlock_targets(job_config):
         return
     if job_config.get("unlock_on_failure", True):
         log.info('Unlocking machines...')
-        lock_ops.unlock_safe(locked, job_config["owner"], job_config["name"], job_config["job_id"])
+        lock_ops.unlock_many(locked, job_config["owner"])
```

Comment:
Why did we switch to `unlock_many`? It doesn't seem to have any checks before unlocking the nodes.